### PR TITLE
added register page reusable components

### PR DESCRIPTION
**Pull request by kdmontero**
Added reusable components to be used in register page

### CODE REVIEW

File: src/components/elements/RegisterInput.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+interface InputProps extends React.InputHTMLAttributes<HTMLInputElement> {}
+
+const RegisterInput = (props: InputProps) => {
+    return <input {...props} className="rounded-lg p-2 ml-2" required />
```
Comment (ksemilla) [inline src/components/elements/RegisterInput.tsx]: Remove `ml-2`. Generally, we don't put padding or margin on just one side of a component. If needed, put the component inside a div then put `ml-2` in that div.

**Review by ksemilla — COMMENTED** [review]
Kindle discuss with me the use of `RegisterSpan`

File: src/components/elements/RegisterInput.tsx (hunk 1 above)
Comment (kdmontero) [reply src/components/elements/RegisterInput.tsx]: done adding an outer div for the `ml-2`

**Comment by kdmontero** [comment]
RegisterSpan is the container of RegisterInput and RegisterLabel. I have another on going branch "register-page" that uses these components.

**Pull request merged**